app/src/main/java/com/ciandt/dragonfly/example/features/projectselection/ProjectSelectionActivity.kt

```diff
@@ -11,33 +11,34 @@ import android.view.View.INVISIBLE
 import android.view.View.VISIBLE
 import com.ciandt.dragonfly.data.model.Model
 import com.ciandt.dragonfly.example.R
-import com.ciandt.dragonfly.example.data.remote.RemoteModelService
+import com.ciandt.dragonfly.example.data.remote.RemoteProjectService
 import com.ciandt.dragonfly.example.features.about.AboutActivity
 import com.ciandt.dragonfly.example.features.realtime.RealTimeActivity
+import com.ciandt.dragonfly.example.models.Project
 import com.ciandt.dragonfly.example.shared.BaseActivity
-import kotlinx.android.synthetic.main.activity_model_selection.*
+import kotlinx.android.synthetic.main.activity_project_selection.*
 
-class ModelSelectionActivity : BaseActivity(), ModelSelectionContract.View {
+class ProjectSelectionActivity : BaseActivity(), ProjectSelectionContract.View {
 
-    private val MODELS_BUNDLE = "MODELS_BUNDLE"
+    private val PROJECTS_BUNDLE = "PROJECTS_BUNDLE"
 
-    private lateinit var presenter: ModelSelectionContract.Presenter
+    private lateinit var presenter: ProjectSelectionContract.Presenter
 
-    private val models = ArrayList<Model>()
+    private val projects = ArrayList<Project>()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_model_selection)
+        setContentView(R.layout.activity_project_selection)
 
-        startService(RemoteModelService.createIntent(this))
-        
-        presenter = ModelSelectionPresenter(ModelSelectionInteractor())
+        startService(RemoteProjectService.createIntent(this))
```

Comment:
What do you think about:

`RemoteProjectService.start(this)` and `RemoteProjectService.stop(this)`

Start/Stop methods in Companion object like this:
```
fun start(context: Context) {
     val intent = Intent(context, RemoteProjectService::class.java)
     context.startService(intent)
 }
```